tests/blavaan_examples.R

```diff
@@ -37,5 +37,8 @@ attr(fitstan@external$mcmcout, 'stanmodel') <- NULL
 attr(fitstanc@external$mcmcout, 'stanmodel') <- NULL
 attr(fitstanfs@external$mcmcout, 'stanmodel') <- NULL
 
-save(list=c("fitjags", "fitstan", "fitstanc", "fitstanfs"), file="../inst/testdata/sysdata.rda")
+save(
+  list=c("fitjags", "fitstan", "fitstanc", "fitstanfs"), 
+  file="inst/testdata/sysdata.rda"
```

Comment:
I'm not sure, but I am guessing we will need the `../` at the start of the file argument